doc/Journal.tex

```diff
@@ -220,143 +220,113 @@ \section{Background}
 The modeled NaI detector includes a 1/8 inch 2024 Aluminum alloy sleeve on which the acrylic RSM is placed. 
-The maximum diameter of the RSM depends on the methodology, but the maximum mask thickness is a constant 7.87 inches (20cm).  
+The maximum diameter of the RSM depends on the methodology, but the maximum mask thickness is a constant 7.87 inches (20 cm).  
 A sphere of air surrounds the source and detector.  
 To increase the solution convergence rate, particles were emitted within a $27.26^\circ$ half angle cone extending from the source to the detector's center.  
 This variance reduction technique assumes that the effect of the few particles that 
-scatter in the air outside of the cone, though the mask, and into the detector is negligible.  
+scatter in the air outside of the cone, though the mask, and into the detector will have negligible contributions to the simulated DRCs.   
 In addition, a 0.095 inch air gap between the mask and aluminum sleeve constrains the mask geometry from impinging on the sleeve and provides a space for grease to be applied between the moving parts.
-Also, note that due to manufacturing constraints each mask angle has a non-zero thickness.
-
-\section{Design Assumptions}
-Monte Carlo N-Particle (MCNP) is one method used to simulate energy spectrum.  
-Logan et al.\cite{Logan2017} showed statistical agreement between experimental and simulated DRCs.  
-Thus, this work will use simulated DRCs to evaluate the RSM performance.  
-Instead of the full energy peak (FEP), the DRC is formed by adding all counts above 200keV.  
-This value was chosen as Logan et al. noted discrepancies due to scatter in the enclosing environment for counts below approximately 200keV.
-
-Herein, it is assumed that the geometry is related to the DRC.  
-Qualitative studies of this correlation showed that in general, this assumption is valid with two qualifications.  
-First, a discontinuous geometry results in a continuous DRC.  
-To understand the next qualification, define a cell to be a rectangular prism with a given thickness extending from the centroid of the detector (outside of the Aluminum sleeve) in a given direction.
+Finally, due to manufacturing constraints, each mask angle must have a non-zero thickness.
+
+\subsection{Design Assumptions and Limitations}
+% This paragraph in general has me a bit confused.  I added the "detector-mask-source" geometry as I think this is what was being referred to.   
+It is assumed that the detector-mask-source geometry is related to the DRC and that geometry can be reconstructed using the DRC.
+Qualitative studies of this correlation showed that, in general, this assumption is valid with two qualifications.  
+%I am not sure how the first qualificiation fits into the preceeding statements.  Is the point that in contrast to Fitzgerald's design assumptions?
```

Comment:
No, rather it is an observation that even though we are using a discrete geometry, the DRC does not show these abrupt changes, but rather smooths them out so they have a more continuous appearance.

Reply:
Understood and edited accordingly